.github/workflows/close-no-repro.yml

```
@@ -0,0 +1,27 @@
+name: Close old issues with no-reproduction label
+
+on:
+    schedule:
+        # 08:00 every monday
+        - cron: 0 8 * * 1
+
+jobs:
+    find-issues:
+        name: Search and destroy
+
+        runs-on: ubuntu-latest
+        permissions:
+            issues: write
+            pull-requests: write
+        steps:
+            - uses: actions/stale@v5
+              with:
+                  only-issue-labels: ⁉️ no-reproduction
+                  days-before-issue-stale: 2
```

Dette skal kanskje være 21, ref. teksten under?

```suggestion
                  days-before-issue-stale: 21
```

---

Oi, her har Petter Pølsefinger vært på ferde